tasks/lint-all.js

```diff
@@ -4,11 +4,118 @@ var gpii = fluid.registerNamespace("gpii");
 
 require("../index");
 require("../src/load-npm-tasks-properly");
+require("gpii-glob");
 
 module.exports = function (grunt) {
     gpii.grunt.lintAll.fixGruntTaskLoading(grunt);
 
-    grunt.config.merge({
+    gpii.grunt.lintAll.mergeAndExpandOptions = function (grunt) {
+        var sanelyMergedOptions = {};
```

Comment:
Odd function global that doesn't seem to be used

Reply:
Skip a bit, brother.  It's used [at the end of the same file](https://github.com/the-t-in-rtf/gpii-grunt-lint-all/blob/GPII-3295/tasks/lint-all.js#L247).  I'm realising it's important enough to document better, I'll add JSDocs.

Reply:
It also needs to be moved out of the module.exports block, which I'll take care of while I'm there.

Reply:
I insist that it is not used except for the momentary assignment 5 lines down

Reply:
The coverage report has a few untested branches that I will add tests to address, but there are no functions that aren't hit by the tests.

https://the-t-in-rtf.github.io/coverage-reports/gpii-grunt-lint-all/20180919/

Reply:
I was interpreting as "global function" rather than "function global", we chatted to clarify.  I have revised to eliminate the function global.